packages/formatter-excel/README.md

```diff
@@ -0,0 +1,20 @@
+# Formatter excel (`@sonarwhal/formatter-excel`)
+
+The `excel` formatter outputs the results in an Excel file, each resource
```

Comment:
Maybe specify what [Excel file format](https://support.office.com/en-us/article/file-formats-that-are-supported-in-excel-0943ff2c-6014-4e8d-aaea-b83d51d46247)?